includes/User.php

```diff
@@ -3949,7 +3949,7 @@ public function getEditToken( $salt = '', $request = null ) {
 			$request = $this->getRequest();
 		}
 
-		if ( $this->isAnon() ) {
+		if ( $this->isAnon() && session_status() !== PHP_SESSION_ACTIVE /* Wikia change (SUS-20) */ ) {
```

Comment:
@Grunny / @wladekb 

MW returns `+\` edit token for anons. Let's generate a token that is less likely to be guessed when an anon has the PHP session (i.e. it can be used to store the token).